web/_templates/team.html

```diff
@@ -0,0 +1,54 @@
+<link rel="stylesheet" href="_static/css/team.css">
```

Comment:
it may help to leave a comment at the top of these templates, that they are autogenerated, should not be edited manually, and that users should edit the json files and run the script instead?

Reply:
That's a great idea. Done!